src/components/Button.js

```diff
@@ -2,13 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const { name, wide, color } = props;
+  const buttonStyle = {
+    width: `${wide === true ? 50 : 25}%`,
```

Comment:
- [ ] OPTIONAL - You can simply do this instead ``${wide ? 50 : 25}%``.

Reply:
Will do that. Thanks